src/components/FileRepo/DownloadFileButton.js

```diff
@@ -28,26 +27,32 @@ const downloadFile = async ({ kfId, api }) => {
   return downloadFileFromGen3({ fileUUID, api });
 };
 
-const DownloadFileButton = compose(injectState, withTheme, withApi)(
+const DownloadFileButton = compose(
+  injectState,
+  withTheme,
+  withApi,
+)(
   ({
     kfId,
     theme,
     effects: { setToast },
     state: { integrationTokens },
     gen3Key = integrationTokens[GEN3],
     api,
+    render,
```

Comment:
A bit conflicting with naming for React's `render` method, but I understand.

Reply:
react's own doc of the pattern actually uses "render" for its example :P